src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -21,6 +22,40 @@ public partial class Popup : PopupBase
 		/// </remarks>
 		internal UIElement Anchor { get; set; }
 
+		/// <summary>
+		/// Returns true if the popup should show the light-dismiss overlay with its current configuration, false if not
+		/// </summary>
+		private bool ShouldShowLightDismissOverlay
+		{
+			get
+			{
+				switch (LightDismissOverlayMode)
+				{
+					case LightDismissOverlayMode.Auto:
+						return false; //TODO: establish sensible platform defaults
```

Comment:
Every time a "TODO" is added it's not related to an Issue or a task?

Reply:
This was a TODO for the current PR, it's been updated